### DISCUSSION TITLE
TextEdit: Bring up keyboard on mobile

### DISCUSSION THREAD

**Discussion by avenger0815** [emilk/egui#764] [Q&A]
Hi, is there a way to automatically bring up the keyboard on a mobile device when an e.g. TextEdit widget is selected?
(I did create an apk for my Android device and the keyboard won't show when clicking into a TextEdit widget)

**Comment by buxx** [comment]
I am also interested in the answer to this question

**Comment by 90-008** [comment]
AFAIK, this needs support in `winit`. I believe https://github.com/rust-windowing/winit/issues/1823 is needed for this to work?

**Comment by Barugon** [comment]
Has there been any progress on this? Now that I have pinch-zoom and long press working, this is the last hurdle to getting an app fully working on a PinePhone.

**Reply by Barugon** [reply]
This now works on Phosh devices.
